### ISSUE TITLE
Pyriform head

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9838]
**Preferred term label:**
Pyriform head

**Synonyms**

**Definition (free text, please give PubMed ID)**
For illustration see: WHO laboratory manual for the examination and processing of human semen (6th Edition)

**Parent term (use hpo.jax.org/app)**
HP:0012865

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 399808

**Your nano-attribution (ORCID)**
0000-0002-7651-2403

**Comment by pnrobinson** [comment]
@MagoWyr  can we use this definition?

Pyriform spermatozoa consist of a normally shaped acrosomal structure which can be normal in size or enlarged, while the post-acrosomal region shows severe elongation and an acute narrowing to the posterior end, ending in about the same thickness as the connecting mid-piece. 
PMID:21076438

**Comment by MagoWyr** [comment]
Yes, that's correct. Thanks for looking up this definition!

**Comment by pnrobinson** [comment]
added as Pyriform sperm head
